docs/notebooks/03_merge.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>You can add function definitions to the python package that is part of the repo (<code>src/atlas_protocol_scripts</code>).  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
If this were just to be published as notebooks, I'd agree. But for the print version it should be more concise while still executable.

Reply:
<p>Also I have been wondering if there is a smarter way of "protecting" integers to be converted to floats when using <code>pd.merge.</code> Currently, <code>dtype={'Start': object, 'End': object}</code> has been my solution ...</p>

Reply:
<p>integers don't support NAs, I guess that's why pandas reads in the column as float by default. <code>object</code> works, but should be avoided in general because it has bad performance. </p><p> </p><p>The proper solution to nullable integers is the [<a href="https://pandas.pydata.org/docs/user_guide/integer_na.html" target="_blank">Int64 type</a>].   </p><p>In this particular case I think you could just keep floats without issues.   </p><p> </p>